src/main/java/decodes/tsdb/algo/ExpressionParserAlgorithm.java

```diff
@@ -1,61 +1,47 @@
 package decodes.tsdb.algo;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.Date;
-import java.util.Observable;
-import java.util.Observer;
-import java.util.Properties;
-
-import org.nfunk.jep.SymbolTable;
```

Comment:
and this still compiles?

Reply:
I used my IDE to cleanup the imports, import org.nfunk.jep.SymbolTable; is still included.